FootprintIOS/FootprintIOS/Sources/View/Info/InfoView.swift

```diff
@@ -0,0 +1,388 @@
+        lb.text = "생년월일"
+        return lb
+    }()
+    
+    private let birthDatePicker: UIDatePicker = {
+        let dp = UIDatePicker()
+        dp.preferredDatePickerStyle = .wheels
+        dp.datePickerMode = .date
+        return dp
+    }()
+    
+    private let bodyLabel: UILabel = {
+        let lb = UILabel()
+        lb.text = "키/몸무게"
+        return lb
+    }()
+    
+    private lazy var bodyInfoButton: UIButton = {
+        let btn = UIButton()
+        btn.setImage(FootprintIOSAsset.Images.bodyInfoButtonIcon.image, for: .normal)
+        return btn
+    }()
+    
+    private let bodyInfoImageView: UIImageView = {
+        let iv = UIImageView()
+        iv.image = FootprintIOSAsset.Images.bodyInfoImage.image
+        //iv.isHidden = true
+        return iv
+    }()
+    
+    private let bodyInfoLabel: UILabel = {
+       let lb = UILabel()
+        lb.text = "칼로리 계산에 이용돼요!\n(미입력시 부정확할 수 있어요)"
+        lb.numberOfLines = 0
+        lb.textColor = .white
+        lb.font = .systemFont(ofSize: 12, weight: .medium)
+        lb.textAlignment = .center
+        return lb
+    }()
+    
+    private let bodyInfoStackView: UIStackView = {
+        let sv = UIStackView()
+        sv.spacing = 40
+        sv.axis = .horizontal
+        sv.alignment = .center
+        return sv
+    }()
+    
+    private let heightLabel: UILabel = {
+        let lb = UILabel()
+        lb.text = "cm"
+        lb.textColor = FootprintIOSAsset.Colors.blueL.color
+        lb.font = .systemFont(ofSize: 16)
+        return lb
+    }()
+    
+    private let heightTextField: UITextField = {
+        let tf = UITextField()
+        tf.placeholder = "170"
+        return tf
+    }()
+    
+    private let weightLabel: UILabel = {
+        let lb = UILabel()
+        lb.text = "kg"
+        lb.textColor = FootprintIOSAsset.Colors.blueL.color
+        lb.font = .systemFont(ofSize: 16)
+        return lb
+    }()
+    
+    private let weightTextField: UITextField = {
+        let tf = UITextField()
+        tf.placeholder = "50"
+        return tf
+    }()
+    
+    private var infoLabels: [UILabel] = .init()
+    private var infoTextFields: [UITextField] = .init()
+    
+    private lazy var bottomButton: UIButton = FootprintButton.init(type: .next)
+    
+    // MARK: - Initializer
+    
+    init() {
+        super.init(frame: .zero)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Methods
+    
+    override func setupProperty() {
+        super.setupProperty()
+        
+        pageStackView.addArrangedSubview(selectedPageCircle)
+        pageStackView.addArrangedSubview(unSelectedPageCircle)
+        
+        infoLabels = [nicknameLabel, genderLabel, birthLabel, bodyLabel]
+        infoLabels.forEach {
+            $0.textColor = .black
+            $0.font = .systemFont(ofSize: 16, weight: .semibold)
+        }
```

Comment:
오, , , 별거 아닌것 같아도 이렇게 하면 코드가 꽤 많이 줄겠네용 
멋져요 좋아요 굿굿

Reply:
하하 감사합니당 ㅎㅎ ❤️
기본 PR 템플릿 보면 <img src = "" width ="250"> 이렇게 되어 있을텐데 
저기 img src에 이미지 링크 넣어주세요 ~!

Reply:
infoLabels변수가 단순히 프로퍼티를 정의하기 위함이라면 ? (== 한번만 쓰이고 안쓰일 변수)
선언할 필요없이 다음과 같이 작성하는게 좋아보이는데 어떻게 생각하시나요 . ?

```suggestion
        [nicknameLabel, genderLabel, birthLabel, bodyLabel].forEach {
            $0.textColor = .black
            $0.font = .systemFont(ofSize: 16, weight: .semibold)
        }
```

Reply:
좋네요 .. 바로 바꾸러갑니다 ..